.azure/install-cmake.yml

```diff
@@ -0,0 +1,43 @@
+# This template installs CMake (if doesn't exist in the systems)
```

Comment:
This shouldn't be necessary.  Remember that we made that Docker image with Cmake and latest Rust on it?

Or is this way easier?